src/main/java/core/basesyntax/strategy/impl/PurchaseOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class PurchaseOperation implements OperationHandler {
+    @Override
+    public void handle(Storage storage, FruitTransaction transaction) throws RuntimeException {
```

Comment:
According to the checklist, we should not ignore exceptions in try-catch blocks. However, the method signature here suggests that a RuntimeException might be thrown, but it is not caught within this method. It's better to handle the exception where it occurs, providing an informative message and rethrowing it if necessary.